aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecCodeSpan.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2016 Automattic
+ * Copyright (C) 2015 Matthew Lee
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wordpress.aztec.spans
+
+import android.graphics.Typeface
+import android.os.Parcel
+import android.text.ParcelableSpan
+import android.text.TextPaint
+import android.text.TextUtils
+import android.text.style.CharacterStyle
+import android.text.style.UpdateAppearance
+import org.wordpress.aztec.formatting.InlineFormatter
+
+class AztecCodeSpan : CharacterStyle, UpdateAppearance, ParcelableSpan, AztecContentSpan, AztecInlineSpan {
```

Comment:
Hey @0nko so, was under the impression that that's what we need to do when we have local variables, like the colors. I got that impression from `AztecQuoteSpan`.

Judging from your question, I now presume that implementing the `ParcelableSpan` might have some other reasoning. Can you elaborate if so? Thanks!